component/src/main/java/com/celements/structEditor/fields/HiddenTagPageType.java

```diff
@@ -42,9 +42,18 @@ public void collectAttributes(AttributeBuilder attrBuilder, DocumentReference ce
     attrBuilder.addNonEmptyAttribute("type", "hidden");
     try {
       XWikiDocument cellDoc = modelAccess.getDocument(cellDocRef);
-      attrBuilder.addNonEmptyAttribute("name", modelAccess.getFieldValue(cellDoc, FIELD_NAME).or(
-          cellDocRef.getName()));
-      attrBuilder.addNonEmptyAttribute("value", getVelocityFieldValue(cellDoc, FIELD_VALUE).or(""));
+      String value = "";
+      String name = "";
```

Comment:
fyi: you needn't always initialise a variable. a simple declaration `String name;` is fine as long as it's initialised in the following if or else block.